### PR TITLE
Fixes for analyzers documentation

### PR DESCRIPTION
**Pull request by alexbakharew**
ArangoDB PR: https://github.com/arangodb/arangodb/pull/16571

### CODE REVIEW

File: scripts/migrate.js [hunk 1]
```diff
@@ -155,6 +155,7 @@ async function migrateMds(basePath, targetPath) {
         });
         content = content.replace(/^\s*@startDocuBlock\s+program_options_(\w+).*$/mg, "{% assign options = site.data[\"" + version.replace('.', '') + "-program-options-\$1\"] %}{% include program-option.html options=options name=\"\$1\" %}");
         content = content.replace(/^\s*@startDocuBlock\s+(\w+).*$/mg, "{% docublock \$1 %}")
+        content = content.replace(/^\s*tests\/js\/common\/aql\/iresearch\/model_cooking.bin.*$/mg, "/path_to_local_fasttext_model_directory/model_cooking.bin")
```
Comment (Simran-B) [inline scripts/migrate.js]: This script was used for a one-off migration from Gitbook to Jekyll and should thus not be modified.

**Comment by alexbakharew** [comment]
@Simran-B @KVS85 Ready for review

**Comment by KVS85** [comment]
Drafted for possible future usage.

**Comment by Simran-B** [comment]
@ansoboleva Should be closed IMO

**Comment by Simran-B** [comment]
No progress has been made regarding an improvement of the demo fastText model that would allow us to actually generate examples with it.

**Pull request closed without merging**